src/js/controller.js

```diff
@@ -564,8 +564,12 @@ export const MediaUIStates = {
       return Array.from(chaptersTrack?.cues  ?? [])
         .map((/** @type VTTCue */{ text, startTime, endTime }) => ({ text, startTime, endTime }));
     },
-    mediaEvents: ['loadstart', 'loadedmetadata'],
-    textTracksEvents: ['addtrack', 'removetrack'],
+    mediaEvents: ['loadstart'],
+    textTracksEvents: ['addtrack', 'removetrack', 'change'],
```

Comment:
I added `change` as well because it makes sense to re-render when the active chapters track changes

Reply:
cool def makes sense to me.